src/diffusers/loaders/lora_pipeline.py

```diff
@@ -1703,6 +1708,7 @@ def save_lora_weights(
             weight_name=weight_name,
             save_function=save_function,
             safe_serialization=safe_serialization,
+            lora_adapter_metadata=lora_adapter_metadata,
```

Comment:
There are some instances of `cls.write_lora_layers` in this file where this new argument has not been propagated, I'm not sure if it's intended or not.

Reply:
Yeah that is intentional as I have not propagated that to every other pipeline.